leasing/report/lease/rent_type.py

```diff
@@ -10,21 +11,25 @@
 from leasing.report.report_base import ReportBase
 
 
-def get_lease_id(obj):
+class RentWithContactNames(Protocol):
```

Comment:
If this protocol was used elsewhere, should it be instead be defined in models>rent.py, exported in the __init__.py and imported in this file? Currently there probably is no need for that, as it's a new class with no other uses.

Reply:
It is only used here, and for typing purposes.
https://github.com/City-of-Helsinki/mvj/pull/687/files#diff-caa21fe3b2c4a67498accf49eea6c4dcb3268bb87cb38d162030de3a6045b847R167 adds `contact_names` to the queryset by `annotate()`, so it won't appear anywhere else.